server/src/main/java/com/objectcomputing/checkins/services/team/TeamServicesImpl.java

```diff
@@ -42,7 +42,7 @@ public TeamResponseDTO save(TeamCreateDTO teamDTO) {
         Team newTeamEntity = null;
         List<TeamMemberResponseDTO> newMembers = new ArrayList<>();
         if (teamDTO != null) {
-            if (teamsRepo.findByName(teamDTO.getName()).isPresent()) {
+            if (!teamsRepo.search(teamDTO.getName(), null).isEmpty()) {
```

Comment:
we can use .isPresent() instead of Not-of-isEmpty() 🙊

Reply:
for some reason .isPresent is not a option there